libs/components/forms/src/lib/modules/input-box/input-box.component.html

```diff
@@ -88,34 +88,45 @@
 
 <ng-template #inputTemplate>
   <ng-content select=".sky-form-control"></ng-content>
-  <ng-container *ngTemplateOutlet="hostInputTemplate"></ng-container>
+  <ng-container *ngIf="hostInputTemplate">
+    <ng-container *ngTemplateOutlet="hostInputTemplate"></ng-container
+  ></ng-container>
 </ng-template>
 
 <ng-template #buttonsLeftTemplate>
   <ng-content select=".sky-input-group-btn.sky-input-box-btn-left"></ng-content>
-  <ng-container *ngTemplateOutlet="hostButtonsLeftTemplate"></ng-container>
+
```

Comment:
Can you remove this extra newline just in case it interferes with the inner content of input box fields.

Reply:
Removed